vector/src/main/java/im/vector/app/features/contactsbook/ContactsBookFragment.kt

```diff
@@ -98,6 +115,7 @@ class ContactsBookFragment @Inject constructor(
     }
 
     override fun invalidate() = withState(contactsBookViewModel) { state ->
+        phoneBookSearchForMatrixContacts.isVisible = state.filteredMappedContacts.isNotEmpty() && state.identityServerUrl != null && !state.userConsent
```

Comment:
Why it is invisible if `!state.userConsent`?

Reply:
It is **visible** if `!state.userConsent` :)
The button is displayed only if the user have not given their consent yet.